Src/BCITraining.cs

```diff
@@ -250,7 +250,7 @@ private void OnQueryProfileOK(object sender, List<string> profiles)
                 if (!string.IsNullOrEmpty(_wantedProfileName))
                 {
                     // create new profile
```

Comment:
does it support for multi headset connected?

Reply:
Not for now @hoangphamemotiv , Even EmotivBCI also support 1 headset at same time when work with profile

Reply:
but the plugin is written to work for all cases. so, multi headset is a case need to be handled

Reply:
@hoangphamemotiv , to support multiple headset it will make it quite complicated and out of scope of this PR. I also haven't seen any request to support multiple headset when work with profile from community users. So I think we should keep as current

Reply:
so a @tungntEmotiv . in case we haven't supported it yet. Could you add a description for note about this issue? so, we can come back later